sdk/Lusid.Sdk.Tests/tutorials/Instruments/CreditDefaultSwapPricing.cs

```diff
@@ -0,0 +1,90 @@
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            _apiFactory = TestLusidApiFactoryBuilder.CreateApiFactory("secrets.json");
+            _testDataUtilities = new TestDataUtilities(
+                _apiFactory.Api<ITransactionPortfoliosApi>(),
+                _apiFactory.Api<IInstrumentsApi>(),
+                _apiFactory.Api<IQuotesApi>(),
+                _apiFactory.Api<IComplexMarketDataApi>());
+        }
+        
+        public void TestDemonstratingThePricingOfCreditDefaultSwaps()
+        {
+            // CREATE a portfolio with instrument
+            var scope = Guid.NewGuid().ToString();
+            var portfolioId = _testDataUtilities.CreateTransactionPortfolio(scope);
+            LusidInstrument instrument = InstrumentExamples.CreateExampleCreditDefaultSwap();
+            
+            // UPSERT the above instrument to portfolio as well as populating stores with required market data
+            _testDataUtilities.AddInstrumentsTransactionPortfolioAndPopulateRequiredMarketData(
+                scope, 
+                portfolioId,
+                TestEffectiveAt,
+                TestEffectiveAt,
+                instrument);
+            
+            // UPSERT CDS spread curve before upserting recipe
+            var cds = (CreditDefaultSwap) instrument;
+            _testDataUtilities.UpsertCdsSpreadCurves(scope, TestEffectiveAt, cds.Ticker, cds.FlowConventions.Currency, cds.ProtectionDetailSpecification.Seniority, 
+                cds.ProtectionDetailSpecification.RestructuringType);
+
+            // CREATE and upsert recipe specifying discount pricing model
+            var discountingRecipeCode = "DiscountingRecipe";
+            CreditDefaultSwapCashFlow.CreateAndUpsertRecipe(discountingRecipeCode, scope, ModelSelection.ModelEnum.Discounting);
+
+            // CREATE valuation request
+            var valuationSchedule = new ValuationSchedule(effectiveAt: TestEffectiveAt);
+            var valuationRequest = new ValuationRequest(
+                recipeId: new ResourceId(scope, discountingRecipeCode),
+                metrics: ValuationSpec,
+                valuationSchedule: valuationSchedule,
+                sort: new List<OrderBySpec> {new OrderBySpec(ValuationDateKey, OrderBySpec.SortOrderEnum.Ascending)},
+                portfolioEntityIds: new List<PortfolioEntityId> {new PortfolioEntityId(scope, portfolioId)},
+                reportCurrency: "USD");
+
+            // CALL valuation
+            var valuation = _apiFactory.Api<IAggregationApi>().GetValuation(valuationRequest);
+            Assert.That(valuation, Is.Not.Null);
+            Assert.That(valuation.Data.Count, Is.EqualTo(1));
+
+            // CHECK PV - note that swaps/forwards can have negative PV
+            var pv = (double) valuation.Data.First()[HoldingPvKey];
+            Assert.That(pv, Is.Not.Null);
+            Assert.That(pv, Is.GreaterThanOrEqualTo(0));
```

Comment:
think we don't want this assertion either.

Reply:
Removed the second one, kept the one checking that PV is not null.